tests/unit/utils/test_pipeline_docker_image_builder.py

```diff
@@ -38,6 +38,29 @@ def test_including_active_profile_in_build_context(tmp_path: Path):
     assert not config_path.exists()
 
 
+def test_check_user_is_set():
+    config = DockerConfiguration(
+        install_stack_requirements=False,
+        requirements=None,
+        required_integrations=[],
+        user=None,
+        replicate_local_python_environment="pip_freeze",
+    )
+    generated_dockerfile = PipelineDockerImageBuilder._generate_zenml_pipeline_dockerfile("test:test", config)
+    assert all(["USER" not in line for line in generated_dockerfile])
+
+    config = DockerConfiguration(
+        install_stack_requirements=False,
+        requirements=None,
+        required_integrations=[],
+        user="test_user",
+        replicate_local_python_environment="pip_freeze",
+    )
+    generated_dockerfile = PipelineDockerImageBuilder._generate_zenml_pipeline_dockerfile("test:test", config)
+    print(generated_dockerfile)
```

Comment:
Not sure if the print here is neccessary?

Reply:
Agreed. Removed